railties/test/commands/routes_test.rb

```diff
@@ -239,20 +239,25 @@ class Rails::Command::RoutesTest < ActiveSupport::TestCase
       end
     RUBY
 
-    output = IO.stub(:console_size, [0, 27]) do
+    lines = IO.stub(:console_size, [0, 27]) do
       run_routes_command([ "--expanded" ])
-    end
+    end.lines
 
     rails_gem_root = File.expand_path("../../../../", __FILE__)
 
-    # rubocop:disable Layout/TrailingWhitespace
-    assert_equal <<~MESSAGE, output
+    propshaft_route = lines[0..6].join
+    remaining_routes = lines[6..].join
```

Comment:
I think this could create some weird edge case when the output changes again, or propshaft adds/removes routes.

Can you think of another way to filter the routes?

Thanks for working on this!

Reply:
Took a peek at how propshaft does it and the presence of routes relies on `config.assets.server`. So setting that to false skips drawing them which makes this change much less brittle